openpype/hosts/fusion/deploy/fusion_shared.prefs

```diff
@@ -0,0 +1,19 @@
+{
+Locked = true,
+Global = {
+ Paths = {
+  Map = {
+   ["OpenPype:"] = "$(OPENPYPE_FUSION)/deploy",
+   ["Reactor:"] = "$(REACTOR)",
```

Comment:
Could not find it. Is there set somewhere in app environments?

Reply:
Ah, REACTOR is only there because it's used by almost everyone working with Fusion in a more than average capacity. The unfortunate side effect of masterprefs is that it "locks" the settings to those values. So a user/studio couldn't set custom path maps for their studio unless they were to change the `.prefs` file.

`REACTOR` would be an environment variable set by the studio as to where they have that installed (if they use it). If they don't it's mostly disregarded by fusion because the pathmapping doesn't resolve to anything.

You'd set it in Admin Settings, like `FUSION16_PYTHON36_HOME` as in my screenshot [here](https://github.com/pypeclub/OpenPype/pull/2769#issuecomment-1049985998).